### PR TITLE
NETOBSERV-792: flag recent metrics as "inaccurate"

### PR DESCRIPTION
**Pull request by jpinsonneau**
## Description

This PR show current "max_chunk_age" value in time picker and query summary side panel.
image

## Dependencies

n/a

## Checklist

If you are not familiar with our processes or don't know what to answer in the list below, let us know in a comment: the maintainers will take care of that.

* [X] Is this PR backed with a JIRA ticket? If so, make sure it is written as a title prefix _(in general, PRs affecting the NetObserv/Network Observability product should be backed with a JIRA ticket - especially if they bring user facing changes)._
* [ ] Does this PR require product documentation?
  * [ ] If so, make sure the JIRA epic is labelled with "documentation" and provides a description relevant for doc writers, such as use cases or scenarios. Any required step to activate or configure the feature should be documented there, such as new CRD knobs.
* [ ] Does this PR require a product release notes entry?
  * [ ] If so, fill in "Release Note Text" in the JIRA.
* [ ] Is there anything else the QE team should know before testing? E.g: configuration changes, environment setup, etc.
  * [ ] If so, make sure it is described in the JIRA ticket.
* QE requirements (check 1 from the list):
  * [X] Standard QE validation, with pre-merge tests unless stated otherwise.
  * [ ] Regression tests only (e.g. refactoring with no user-facing change).
  * [ ] No QE (e.g. trivial change with high reviewer's confidence, or per agreement with the QE team).

### CODE REVIEW

File: web/src/components/modals/time-range-modal.tsx [hunk 1]
```diff
@@ -271,6 +279,16 @@ export const TimeRangeModal: React.FC<TimeRangeModalProps> = ({ id, isModalOpen,
         </Text>
       </TextContent>
       {getDateTimePickers()}
+      {!Number.isNaN(maxChunkAge) && (
+        <TextContent>
+          <Text component={TextVariants.blockquote}>
+            {t(
+              'Collection latency could be up to {{maxChunkAge}} corresponding to the current Loki "max_chunk_age" ingester configuration.',
```
Comment (jotak) [inline web/src/components/modals/time-range-modal.tsx:286]: what exactly is the intent of this message on this dialog? Users can select a time range bigger than the max chunk age, right?

Comment (jpinsonneau) [reply web/src/components/modals/time-range-modal.tsx:286]: The goal is to inform users that flows/metrics can still appear during the max chunk age = 2h for loki operator.
According to our last discussion, we decided to put the message in the time frame selection.

File: web/src/utils/duration.ts [hunk 2]
```diff
@@ -7,20 +8,11 @@ const h = m * 60;
...
 
 // Converts a duration like "1h 10m 23s" to milliseconds or throws an error if the duration could not be
 // parsed
 export const parseDuration = (duration: string): number => {
-  const parts = duration
-    .trim()
-    .split(wordsRegexp)
-    .map(p => p.match(durationRegexp));
-  return _.sumBy(parts, p => parseInt(p![1], 10) * units[p![2] as KeyOfUnits]);
+  return parse(duration, 'ms')!;
```
Comment (jotak) [inline web/src/utils/duration.ts:15]: you may have to adapt the new tests that I added , after rebasing

**Review by jotak — APPROVED** [review]
not tested; code LGTM

File: web/src/utils/duration.ts (hunk 2 above)
Comment (jpinsonneau) [reply web/src/utils/duration.ts:15]: Tests seems ok as is !

**Pull request merged**